src/Microsoft.OData.Core/ODataWriterCore.cs

```diff
@@ -1052,6 +1109,8 @@ private void WriteEndImplementation()
                         }
 
                         break;
+                    case WriterState.Primitive:
+                        break;

```

Comment:
>break [](start = 24, length = 5)

Can you add a comment stating your intention, I think there is nothing to do to close a primitive? Just want it to not look like something was accidentally deleted.